codec/seata-codec-seata/src/test/resources/file.conf

```diff
@@ -45,30 +72,67 @@ store {
 
   ## database store
   db {
```

Comment:
only used for seata-server, I think I should remove the configuration of this node.